dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -962,12 +962,12 @@ def create_admin_script
     assert_response :forbidden
   end
 
-  test "should get show of admin script if signed in as admin" do
+  test "should not get show of admin script if signed in as admin" do
```

Comment:
Who should be able to see the admin script, if not admin? This is a bit confusing. Is it meant to be levelbuilder? Maybe rename levelbuilder_script? Or At least add a comment in the test.